app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/MappingWizardDatastoresStep.js

```diff
@@ -109,16 +110,27 @@ class MappingWizardDatastoresStep extends React.Component {
 
     const { selectedCluster, selectedClusterMapping } = this.state;
 
+    const clusterOptions = getClusterOptions(clusterMappings);
+
     // first we render the dropdown selection for each source cluster in clusterMappings,
     // then we call `selectSourceCluster` and go get that cluster's datastores on selection
     return (
       <div>
-        <SourceClusterSelect
-          clusterMappings={clusterMappings}
-          selectSourceCluster={this.selectSourceCluster}
-          selectedCluster={selectedCluster}
-          selectedClusterMapping={selectedClusterMapping}
-          form={form}
+        <Field
+          name="cluster_select"
+          label={__('Map source datastores to target datastores for cluster')}
+          data_live_search="true"
+          component={BootstrapSelect}
+          options={clusterOptions}
+          option_key="id"
+          option_value="name"
+          onSelect={this.selectSourceCluster}
+          pre_selected_value={
+            clusterOptions.length === 1 ? clusterOptions[0].id : ''
+          }
+          choose_text={`<${__('Select a source cluster')}>`}
+          render_within_form="true"
+          form_name={form}
```

Comment:
So, since we're now hooking the selects into `redux-form`,  a couple of other improvements we can make are

1. Lift the `selectedCluster` component state to the redux store
2. We can now use `redux-form` validation to handle the `next` button disabling.  Before I had to use a workaround where I was rendering `DualPaneMapper`, but hiding it [(find hack here)](
https://github.com/priley86/miq_v2v_ui_plugin/blob/master/app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/components/DatastoresStepForm/DatastoresStepForm.js#L373-L375).  We can instead use `length` from `redux-form-validators` and go back to conditionally rendering [this guy](https://github.com/priley86/miq_v2v_ui_plugin/blob/master/app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/MappingWizardDatastoresStep.js#L123-L135)

If we want to make these changes, I imagine it would be best to do it as a separate PR

Reply:
yes - i think lifting `selectedCluster` to redux store is fine (so good w/ #1). 

For #2 (handle next button disabling)... i think that's probably OK but we have to vet this change w/ @mturley 's wizard abstraction. I believe `shouldDisableNextStep(activeStepIndex)` callback is called for a step now, so we'd just have to verify we can still map this to `redux-form-validators`. This should all jive in a future PR... but I'm fine w/ what's been added here...